code/android-core-library/src/main/java/com/adobe/marketing/mobile/rulesengine/download/RulesDownloadCallback.java

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
This contains logic specific to launch. We should move this inside `launchrulesengine` package.